frontend/components/commons/results/RecordExtraActions.vue

```diff
@@ -126,6 +113,16 @@ export default {
     close() {
       this.open = false;
     },
+    copyToClipboard() {
+      if (isObject(this.record.inputs)) {
+        const text = Object.keys(this.record.inputs).map((key) => {
+          return `${key}\r\n${this.record.inputs[key]}`;
+        });
+        return this.$copyToClipboard(text);
+      } else {
+        return this.$copyToClipboard(this.record.text);
```

Comment:
Suggesting a code refactoring, we could move the text generation logic to each record class, keeping the `copyToClipboard` method simpler. Something like:

```javascript
copyToClipboard() {
   return this.$copyToClipboard(this.record.clipboardText); // or some other representative name 
}
```

Then each Record class should implement its own logic to generate the `clipboardText` property. For example, in the case of text classification, you can include the object parsing and all the stuff.